### PR TITLE
website/integrations: add account linking note for WriteFreely

### PR DESCRIPTION
**Pull request by ghost**
Hey everyone,

when connecting authentik with WriteFreely as shown on [https://goauthentik.io/integrations/services/writefreely/](url), I couldn't sign in first as my authentic-user (mail address) was different from the login in WriteFreely (just my name).

## Details

-   **Does this resolve an issue?**
    Resolves missing SSO after setting up authentik for the login workflow in WriteFreely.

### New Features

-   Adds account linking to the documentation.

### Breaking Changes

-   Adds breaking change which causes \<issue\>.

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)
-   [ ] The translation files have been updated (`make i18n-extract`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

Bildschirmfoto 2023-05-29 um 20 20 28

### CODE REVIEW

**Review by BeryJu — APPROVED** [review]
LGTM technically, feel free to check the copy @tanberry

File: website/integrations/services/writefreely/index.md [hunk 1]
```diff
@@ -88,6 +88,12 @@ map_email          = email
 
 Restart writefreely.service
 
+## Account linking
+
+If your username in authentik and your WriteFreely username differ, you may need to link your accounts before being able to use SSO.
```
Comment (tanberry) [inline website/integrations/services/writefreely/index.md:93]:
```suggestion
If your usernames in authentik and WriteFreely are different, you might need to link your accounts before being able to use SSO.
```

File: website/integrations/services/writefreely/index.md [hunk 2]
```diff
@@ -88,6 +88,12 @@ map_email          = email
...
+
+This can be done after logging into Writefreely with local credentials, navigating to "Customize" in the menu and "Account settings" afterwards. Amongst the option "Linked Accounts", click on "authentik", and your accounts will be linked.
```
Comment (tanberry) [inline website/integrations/services/writefreely/index.md:95]:
```suggestion
To link the accounts, first log into Writefreely with local credentials, and then navigate to **Customize -->Account Settings**. In the option "Linked Accounts", click on "authentik".
```

**Review by tanberry — APPROVED** [review]
Thanks so much @OliverPifferi for adding this valuable information to our docs!

**Pull request merged**